components/List/tests/unit/MDCListTypographyThemerTests.m

```diff
@@ -31,4 +69,46 @@ - (void)testApplyingTypographyScheme {
   XCTAssertEqualObjects(cell.detailLabel.font, typographyScheme.body2);
 }
 
+- (void)testFontThemerWhenCurrentContentSizeCategoryIsUsed {
+  if (@available(iOS 10.0, *)) {
+    // Given
+    MDCSelfSizingStereoCell *cell = [[MDCSelfSizingStereoCell alloc] initWithFrame:CGRectZero];
+    MDCTypographyScheme *typographyScheme = [[MDCTypographyScheme alloc] init];
+    typographyScheme.useCurrentContentSizeCategoryWhenApplied = YES;
+
+    // When
+    MDCListTypographyThemerContentSizeCategoryOverrideWindow *extraLargeContainer =
+        [[MDCListTypographyThemerContentSizeCategoryOverrideWindow alloc]
+            initWithContentSizeCategoryOverride:UIContentSizeCategoryExtraLarge];
+    [extraLargeContainer addSubview:cell];
+    [MDCListTypographyThemer applyTypographyScheme:typographyScheme toSelfSizingStereoCell:cell];
+
+    // Then
+    XCTAssertNotNil(cell.titleLabel.font.mdc_scalingCurve);
+    XCTAssertFalse([cell.titleLabel.font mdc_isSimplyEqual:typographyScheme.subtitle1]);
+    XCTAssertFalse([cell.detailLabel.font mdc_isSimplyEqual:typographyScheme.body2]);
```

Comment:
Consider checking the point sizes here to make sure the fonts are growing as expected.

Reply:
Addressed in 9808c7f6acac18d0cf0909c5d6b06f247907fb6b.